docs/plugins/Logs-endpoint-plugin.md

```diff
@@ -0,0 +1,103 @@
+# Logs Endpoint Plugin 
+The Logs Endpoint plugin provides HTPP access to EventStoreDB log files, facilitating easy log management. 
+
+::: tip
+The Logs Endpoint plugin is included in commercial versions of EventStoreDB.
+:::
+
+## Enabling the Plugin
+
+1. Download and Install: First, download the plugin and place it in the `plugins` folder within your EventStoreDB installation directory. 
```

Comment:
```suggestion
1. Download and install: First, download the plugin and place it in the `plugins` folder within your EventStoreDB installation directory. 
```

Reply:
The other docs use sentence case in the quick start steps.

Reply:
Good eye